### PR TITLE
refine fastlane CI for SA

### PR DESCRIPTION
**Pull request by huminzhi**

### CODE REVIEW

**Comment by huminzhi** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          2 Errors
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Code coverage under minimum of 80%</td>
    </tr>
  </tbody>
</table>

### Tests: 

Classname | Name|
--- | ---|
CSFMultipartInputStreamTests | testMixedObjectElements|
CSFMultipartInputStreamTests | testMixedObjectElements|
CSFMultipartInputStreamTests | testMixedObjectElements|
CSFMultipartInputStreamTests | testMixedObjectElements|
CSFMultipartInputStreamTests | testMixedObjectElements|
CSFMultipartInputStreamTests | testMultipleStringElements|
CSFMultipartInputStreamTests | testMultipleStringElements|
CSFMultipartInputStreamTests | testMultipleStringElements|
CSFMultipartInputStreamTests | testMultipleStringElements|
CSFMultipartInputStreamTests | testSingleStringElement|
CSFMultipartInputStreamTests | testSingleStringElement|


## Current coverage for SalesforceAnalytics.framework is `73%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceSDKCore.framework is `51%`
✅ *No files affecting coverage found*

---
## Current coverage for CocoaLumberjack.framework is `32%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartStore.framework is `63%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartSync.framework is `76%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceHybridSDK.framework is `43%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceReact.framework is `0%`
✅ *No files affecting coverage found*

---
## Current coverage for RestAPIExplorer.app is `26%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartSyncExplorer.app is `19%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartSyncExplorerCommon.framework is `0%`
✅ *No files affecting coverage found*

---
## Current coverage for RecentContactsTodayExtension.appex is `0%`
✅ *No files affecting coverage found*

---
## Current coverage for AccountEditor.app is `0%`
✅ *No files affecting coverage found*

---
## Current coverage for NoteSync.app is `0%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartSyncExplorerHybrid.app is `0%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceSDKCoreTestApp.app is `30%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartStoreTestApp.app is `61%`
✅ *No files affecting coverage found*

---
## Current coverage for SmartSyncTestApp.app is `61%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceHybridSDKTestApp.app is `85%`
✅ *No files affecting coverage found*

---
## Current coverage for SalesforceAnalyticsTestApp.app is `61%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_CI-result">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by bhariharan** [comment]
@huminzhi The `default` link here points to an internal test result link. Any way we could get rid of that link? Looks good to me, otherwise.

**Comment by huminzhi** [comment]
@bhariharan the default link is a jenkins setting, I am updating Jenkins setting to remove it, but this PR should be ok

**Comment by bhariharan** [comment]
Thanks @huminzhi

File: fastlane/Fastfile [hunk 1]
```diff
@@ -1,37 +1,63 @@
...
-	   	UI.error(ex)
+		UI.error(ex)
 	end
 
 end
+
+
+lane  :localtest do
+#setup test accounts (for sf dev only)
+	`git clone https://git.soma.salesforce.com/gist/d103e1ef29e1c11ee9a7a4733a1193d8.git`
```
Comment (bhariharan) [inline fastlane/Fastfile]: Can we remove this @huminzhi? It's an internal reference.

**Pull request merged**